srcs/readMesh.cpp

```diff
@@ -173,6 +195,11 @@ bool readMesh(MeshParams& meshParams, const std::string& fileName,
             std::vector<std::pair<int, int>> edgeList;
             std::vector<std::vector<double>> normalList;
             std::vector<double> meanNormal(2,0);
+            std::vector<int> elementIndex;
+            elementIndex.push_back(nEdgePerEl*elm);
+            elementIndex.push_back(nEdgePerEl*elm+1);
+            elementIndex.push_back(nEdgePerEl*elm+2);
```

Comment:
Does that mean that it only works for T3 ? Shouldn't we make a loop instead ?

Reply:
We will look at that later ^^.